ocs_ci/helpers/cephx_keyrotation_helper.py

```
@@ -0,0 +1,836 @@
+        Wait until all Running pods for *label* have new names or annotations.
+
+        Args:
+            before_state (dict): Output of :meth:`capture_daemon_pod_state`.
+        """
+        log.info(
+            "Waiting for pod restarts (label=%s, prior pods=%s)",
+            label,
+            ", ".join(before_state) or "none",
+        )
+
+        def _pods_restarted():
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_pods_restarted`

Add return type annotation: `bool`

(ANN202)